src/Umbraco.Web/Editors/ContentController.cs

```diff
@@ -1040,14 +1049,16 @@ private bool SaveScheduleVariant(ContentItemSave contentItem)
                 if (!isPublished && releaseDates.Count == 0)
                 {
                     //can't continue, a mandatory variant is not published and not scheduled for publishing
-                    AddCultureValidationError(culture, "speechBubbles/scheduleErrReleaseDate2");
+                    // TODO: Add segment
```

Comment:
Do we need this TODO?

Reply:
Yes, if we want to deal with segments then we need to parse the segment not 'null'.

Reply:
(Sorry, I'll edit my comment to this:) If we want to support mandatory segmented variants we should add segments. But currently its not part of the feature therefor it can stay 'null' and we can remove the Todo.